modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/AndesMessageMetadata.java

```diff
@@ -284,6 +284,7 @@ public AndesMessageMetadata shallowCopy(long messageId) {
         clone.retain = retain;
         clone.metadata = metadata;
         clone.expirationTime = expirationTime;
+        clone.isExpirationDefined = isExpirationDefined;
```

Comment:
Not something that has been introduced by this change, but something that can be fixed.

The variable name 'isExpirationDefined' violates best practices. The variable name should be 'expirationDefined'. The getter method for this should be 'isExpirationDefined'.